vmware_dvs/api/dvs_agent_rpc_api.py

```diff
@@ -76,10 +76,12 @@ def update_network_cast(self, current, segment, original):
                                       current=current, segment=segment,
                                       original=original)
 
-    def bind_port_call(self, current, network_segments, network_current, host):
+    def bind_port_call(self, current, network_segments, network_current, host,
+                       vif_details=None):
         return self._get_cctxt_direct(host).call(
             self.context, 'bind_port', current=current,
```

Comment:
You need to update server side API too. But I disagree with RPC update: you can inject info to current parameters. It will be simpler.